package.json

```diff
@@ -49,28 +49,28 @@
     "eslint-plugin-prettier": "^3.0.1",
     "express": "^4.9.8",
     "ghooks": "^0.3.0",
-    "gulp": "^3.8.11",
+    "gulp": "^4.0.0",
     "gulp-babel": "^6.1.3",
-    "gulp-coveralls": "^0.1.3",
+    "gulp-coveralls": "^0.1.4",
     "gulp-eslint": "^5.0.0",
     "gulp-git": "~0.5.3",
-    "gulp-load-plugins": "^0.9.0",
-    "gulp-mocha": "^2.0.1",
+    "gulp-load-plugins": "^1.5.0",
+    "gulp-mocha": "^6.0.0",
     "gulp-plumber": "^1.0.0",
     "gulp-rename": "~1.2.0",
     "gulp-rimraf": "~0.1.0",
-    "gulp-uglify": "~0.3.1",
+    "gulp-uglify": "^3.0.1",
     "gulp-webpack": "^1.3.1",
     "husky": "^1.3.1",
     "istanbul": "~0.3.2",
-    "karma": "^0.13.9",
+    "karma": "3.1.4",
```

Comment:
To @vcarl's point about not narrowing packages, maybe `"karma": ">=3.1.4 <4.0.0"` does the trick.